src/app/cli/CommandLineInterface.java

```diff
@@ -63,15 +92,280 @@ public void launch(){
+
+                    if(collections_list.size() > 1) {
+                        System.out.println("Which collection would you like to remove the game from (enter the number)?");
+                        for(int i = 0; i < collections_list.size(); ++i) {
+                            Collection curr_coll = collections_list.get(i);
+                            System.out.println(i+1 + ".\tName: " + curr_coll.coll_name() + "\n\t# of games: " +
+                                    curr_coll.games().length + "\n\tPlay time: " +
+                                    app.total_playtime_collection(curr_coll) + "\n");
+                        }
+                        input = in.nextLine();
+                        int input_to_int = Integer.parseInt(input);
+                        selected_coll = collections_list.get(input_to_int-1);
+                    }
+
+                    Game[] game_list_init = selected_coll.games();
+
+                    ArrayList<Game> game_list = new ArrayList<>();
+
+                    for (Game game : game_list_init) {
+                        if (game.title().equals(tokens[1])) {
+                            game_list.add(game);
+                        }
+                    }
+
+                    if(game_list.size() == 0) {
+                        System.out.println("There is not a game with that name in your collection.");
+                        continue;
+                    }
+
+                    Game selected_game = game_list.get(0);
+                    if(game_list.size() > 1) {
+                        System.out.println("Which game would you like to remove (enter the number)?");
+                        for(int i = 0; i < game_list.size(); ++i) {
+                            Game curr_game = game_list.get(i);
+                            System.out.println(i+1 + ".\tName: " + curr_game.title() + "\n\tPlay time: " +
+                                    curr_game.playtime());
+                        }
+                        input = in.nextLine();
+                        int input_to_int = Integer.parseInt(input);
+                        selected_game = game_list.get(input_to_int-1);
+                    }
+
+                    Collection updated_coll  = app.collection_remove(selected_coll, selected_game);
+                    System.out.println("Updated collection:");
+                    System.out.println("Name:\t" + updated_coll.coll_name());
+                    System.out.println("# of games\t" + updated_coll.games().length);
+                    System.out.println("Play time:\t" + app.total_playtime_collection(updated_coll));
+                }
+
+                case COLLECTION_DELETE -> {
+                    Collection[] collections_list_init = app.get_collections();
+                    ArrayList<Collection> collections_list = new ArrayList<>();
+                    for (Collection collection : collections_list_init) {
```

Comment:
could/should probably be abstracted to a `getCollectionByName(String name)`, probably put in `App.java`

Reply:
i just see the same for loop like 6 different times and as such putting it in its own function is probably a good idea

putting it in `App` is probably a good idea bc i think the sql statement can just be made that will do this for you, and having the DB do work is better than our code

Reply:
Sounds good, will put in IApp